sklearn/model_selection/tests/test_validation.py

```diff
@@ -126,6 +126,7 @@ def _is_training_data(self, X):
 X = np.ones((10, 2))
 X_sparse = coo_matrix(X)
 y = np.arange(10) // 2
+y2 = np.array([1, 1, 1, 2, 2, 2, 3, 3, 3, 3])
```

Comment:
Can you just add a comment above saying why `y2` is used instead of `y` for the classifier, that is `number of samples containing a particular class is less than the number of folds`